drm/DrmPlane.cpp

```diff
@@ -191,6 +198,22 @@ bool DrmPlane::IsFormatSupported(uint32_t format) const {
          std::end(formats_) || format == DRM_FORMAT_NV12_Y_TILED_INTEL;
 }
 
+bool DrmPlane::IsResolutionSupported(hwc_rect_t display_frame) {
+  std::pair<uint32_t, uint32_t> min = drm_->GetMinResolution();
+  std::pair<uint32_t, uint32_t> max = drm_->GetMaxResolution();
```

Comment:
for Android BM, will get the physical display plan limitation, while we can have big-joiner feature.

Reply:
Android BM, min: 0,  max: 16384,  will add new kernel interface to get plane limitation

Reply:
Please add these info as comments for upper code:
/*
virtio-gpu's min/max width is ***, min/max height is is ***
TODO: i915 min/max width/height is ***, hasn't implemented in i915 kernel driver
*/